### ISSUE TITLE
Path differs in behavior from v3.5.51 to 3.6.48

### ISSUE THREAD

**Issue by erikari** [EsotericSoftware/spine-editor#303]
When upgrading from the older version, the path behaves differently and does not respect the chain scale it had before.

(I tested it and confirm the bug)

See mails 1604f6bd296c7a32 for repro file, and images of the bug.

**Comment by NathanSweet** [comment]
Here is a more distilled repro (3.7):
http://n4te.com/x/3535-path-scaling.spine
This is a weird setup because both the path and constrained bones are under the same parent bone, and the parent bone is scaled non-uniformly.

In 3.5.51 the path spacing was not scaled using the path bone's scale. In 3.6, the path spacing is scaled, as described here:
http://esotericsoftware.com/forum/Scaling-bone-messes-up-animation-order-7487?p=34659#p34659
The runtime change that scales the path spacing in 3.6 is here:
https://github.com/EsotericSoftware/spine-runtimes/commit/8e0972d732d88ac4d4f6e1531fdd793ba6949928

TLDR; it's not actually a bug, it's an odd bone setup with non-uniform change and a known difference between 3.5 and 3.6 with how path spacing is scaled.

Whew.

**Comment by Monkeyzav** [comment]
Hello!
Cool, good to know. I wanted to check your file 3535-path-scaling.spine
<http://n4te.com/x/3535-path-scaling.spine>  but:

[image: Images intégrées 1]

I don't have this futuristic version :D

2018-02-01 16:49 GMT+01:00 Nathan Sweet <notifications@github.com>:

**Comment by NathanSweet** [comment]
Haha, that's a development version. 3.7.09-beta should be able to open it.

**Comment by Monkeyzav** [comment]
Yeah, I just noticed, thanks!

**Comment by Monkeyzav** [comment]
Ok, I see how it works.

So the only way of having a scale on my parent bone (because I want this), which has a path inside, is working ONLY if my constrainted bones chain is outside of my parent bone.

Then I can manipulate properly the parent bone with scale, rotation, etc, knowing that, even if outside of the hierarchy, the bones will always follow my path (constraint in chain scale of course).

**Comment by NathanSweet** [comment]
You can have the constrained bones under the path's bone, but scaling the parent will change the constrained bones' size. If you don't like that, then the answer is to move the constrained bones from under the path bone.
